Formula/flake8.rb

```diff
@@ -27,6 +26,9 @@ def install
 
   test do
     xy = Language::Python.major_minor_version "python3"
-    system "#{bin}/flake8", "#{libexec}/lib/python#{xy}/site-packages/flake8"
+    # flake8 version 3.6.0 is not compliant to flake8 with default settings.
```

Comment:
Irony level is high.

Reply:
You mean my comment or the problem itself :)? I anyways changed the comment to be a little more neutral.